### DISCUSSION TITLE
Feature request? - Default category

### DISCUSSION THREAD

**Discussion by Haimana** [VueTorrent/VueTorrent#2066] [General]
Hello, and congrats for your excellent work.
I am wondering if there is an option to set up a default category when adding new torrents. Is it somewhere and I am missing it?
If is not, please consider this as a feature request. 
Thank you,
Seb

**Comment by Larsluph** [comment]
I think I remember something similar as being already asked in the past and still don't understand how that would be useful?

I mean, you can configure a default download/save path so I'm not sure what else it could serve for...

**Reply by Haimana** [reply]
thanks for reply
IMHO is useful because my torrents are mostly movies and should stay in different folder/category. a default category will spare few clicks (seconds, wasted) :)
many times I'm accessing VueTorrent by phone and at my age this is very annoying.
best regards,
Seb

**Reply by Larsluph** [reply]
If you're using torrent files I would recommend using watch folders instead, however the API doesn't allow to configure a default category on a watch folder so you'll have to do it manually in the config files.
